ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -30,7 +30,16 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net48'">
-		<PackageReference Include="JustArchiNET.Madness" />
+		<Using Include="Microsoft.AspNetCore.Hosting.IHostingEnvironment" Alias="IWebHostEnvironment" />
+		<Using Include="Microsoft.AspNetCore.Hosting.IWebHost" Alias="IHost" />
+		<Using Include="Microsoft.AspNetCore.Hosting.WebHostBuilder" Alias="HostBuilder" />
+		<Using Include="Microsoft.AspNetCore.Mvc" />
+		<Using Include="Microsoft.AspNetCore.Mvc.MvcJsonOptions" Alias="MvcNewtonsoftJsonOptions" />
+		<Using Include="Microsoft.Extensions.DependencyInjection.IMvcCoreBuilder" Alias="IMvcBuilder" />
+		<Using Include="Newtonsoft.Json.Converters" />
+	</ItemGroup>
```

Comment:
Can we make those work *without* including them in csproj? Madness part which applies globally is a good idea because it's literally a polyfill for framework, but those are netf-specific things, and I don't feel entirely safe doing it like this globally in whole ASF.

On the other hand if it's impossible to achieve this with `using` clauses guarded by `#if NETFRAMEWORK` then it stays because it improves code readability :upside_down_face:

Reply:
The only possibility is by adding following snippet at the beginning of the corresponding files:

```csharp
#if NETFRAMEWORK
using MvcNewtonsoftJsonOptions = Microsoft.AspNetCore.Mvc.MvcJsonOptions;
#endif
```

Reply:
That should still be just one-two files then, as those are ASP.NET parts, the added value of including them globally for every cs file across whole ASF isn't worth the risk of accidentally stumbling into some *even worse madness* just because of implicit usings.

I'd say we should move those back to `.cs` file where they're needed, but keeping the aliases like you did as we then have less ifdefs below in the actual code.